repo/packages/P/prometheus/2/resource.json

```diff
@@ -0,0 +1,66 @@
+{
+    "assets": {
+        "container": {
+            "docker": {
+                "pushgateway": "prom/pushgateway:v0.5.2"
+            }
+        }, 
+        "uris": {
+            "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz", 
+            "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.11.0.tar.gz", 
+            "bootstrap-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.42.1/bootstrap.zip", 
+            "executor-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.42.1/executor.zip", 
+            "scheduler-zip": "https://downloads.mesosphere.io/dcosdev-sdk/assets/0.42.1/operator-scheduler.zip", 
+            "svc": "https://arathorec-test.s3.amazonaws.com/prometheus/artifacts/0.1.2-2.3.2/svc.yml", 
```

Comment:
hi @ankitcid you will have to change the URI for the svc.yml to the downloads one.

Reply:
Hi @realmbgl , done the change in 0.1.2-2.3.2 version.